src/URIParser.jl

```diff
@@ -6,19 +6,12 @@ export URI
 export defrag, userinfo, path_params, query_params, isvalid
 export escape, escape_form, escape_with, unescape, unescape_form
 
-import Base: isequal, isvalid, show, print, (==)
-using Compat
+using Unicode
+using Base.Unicode: isascii
 
-if VERSION >= v"0.7.0-DEV.2915"
-    using Unicode
-    using Base.Unicode: isascii
-end
+import Base: isequal, isvalid, show, print, (==)
 
-if VERSION >= v"0.7.0-DEV.3526"
-    _parse(T, s, b) = Base.parse(Int, s, base=b)
-else
-    _parse(T, s, b) = Base.parse(Int, s, b)
-end
+_parse(T, s, b) = Base.parse(Int, s, base=b)
```

Comment:
Instead I think we should just go ahead and replace uses of `_parse` with `parse` with a keyword argument (no need for the Base prefix either) since this was only added here to maintain backwards compatibility with 0.6.